frontend/test/visualizations/components/LineAreaBarRenderer-bar.unit.spec.js

```diff
@@ -57,18 +57,45 @@ describe("LineAreaBarRenderer-bar", () => {
                         onHoverChange: (hover) => {
                             try {
                                 const data = hover.data && hover.data.map(({key, value}) => ({key, value}));
+                                let standardDisplay
+                                let normalizedDisplay
+
+
                                 hoverCount++;
                                 if (hoverCount === 1) {
-                                    expect(data).toEqual([
+                                    standardDisplay = [
                                         {key: "Category", value: "A"},
                                         {key: "Sum", value: 1}
-                                    ]);
+                                    ];
+
+                                    normalizedDisplay = [
+                                        {key: "Category", value: "A"},
+                                        {key: "% Sum", value: "33%"}
+                                    ]
+
+                                    expect(data).toEqual(
```

Comment:
I'm not a huge fan of special casing tests like this. 

This is mergable, but I'd prefer tests to not include logic that can itself be flakey or change with time.

Reply:
Yeah I agree. This seemed like the path of least resistance to getting a test for the behavior without having to refactor the test at large. This existing test is a bit overly clever with mapping through the types so at some point I think we should rewrite the tests to just use standard describe / it blocks per type to make things clearer.